docs: Update README.md (#792)
gluonhq/scenebuilder: README.md

Reviewer: Do we want to advertise [Gluon Desktop](http://gluonhq.com/products/desktop), and
[Gluon CloudLink](http://gluonhq.com/products/cloudlink)? Ideally, SB only works with controls from Gluon Mobile.

[![Gluon](.github/assets/gluon_logo.svg)](https://gluonhq.com)

# Gluon Scene Builder #

[![Build](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml/badge.svg)](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml)
[![License](https://img.shields.io/badge/license-BSD-green)](./LICENSE)

> Scene Builder Kit: 

Reviewer: Can we remove this text? I feel its given that these  badges are for kit.

If we still want to explicitly mention it, I would make it as a header and put the badges below it.

Author: done

[![Gluon](.github/assets/gluon_logo.svg)](https://gluonhq.com)

# Gluon Scene Builder #

> [![Build](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml/badge.svg)](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml)

Reviewer: This workflow is used to verify PRs. The workflow which shows the current state of master branch is the early-access workflow. We may have to change the name of the badge to "build" instead of default name.

Author: changed. It shows now `Early Access: Passing`, which is fine, since it shows that the EA release is green.

### How to build Scene Builder ###

> Scene Builder makes use of the Maven Wrapper to build and run the project. So there is no need to install Maven on the developers machine. To utilize Maven Wrapper, instead of calling `mvn`, one can run `./mvnw` on Linux or macOS or `mvnw` on Windows.

Reviewer 2: Seems a bit verbose... just a small suggestion:
```suggestion
Scene Builder uses Maven Wrapper to build and run the project. Run `./mvnw` on Linux or macOS and `mvnw` on Windows.
```

Before starting the app, all dependencies must be installed locally.
This is achieved by:

`./mvnw install`

Then Scene Builder can be started with Maven:

> `./wmvn javafx:run -f app`

Reviewer 2: ```suggestion
`./mvnw javafx:run -f app`
```

## Scene Builder Kit ##

[![Build](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml/badge.svg)](https://github.com/gluonhq/scenebuilder/actions/workflows/build.yml)
[![Maven Central](https://img.shields.io/maven-central/v/com.gluonhq.scenebuilder/kit)](https://search.maven.org/#search|ga|1|com.gluonhq.scenebuilder)
[![License](https://img.shields.io/github/license/gluonhq/scenebuilder)](https://opensource.org/licenses/GPL-3.0)

> [![javadoc](https://javadoc.io/badge2/com.gluonhq.scenebuilder/kit/javadoc.svg?color=blue)](https://javadoc.io/doc/com.gluonhq.scenebuilder/kit)

Reviewer: I prefer the badges at repository level on the top.

Author: moved

com.oracle.javafx.scenebuilder.app.SceneBuilderApp
```

## Scene Builder components ##

The Scene Builder project has three main components defined by three modules (that is, three Java modules defined in three Maven modules subprojects):

- Scene Builder App
- Scene Builder Kit
- Gluon plugin

### Scene Builder App ###

Contains the JavaFX main application that embeds the Scene Builder Kit, and includes menus, preferences and dialogs to interact with it.

### Scene Builder Kit ###

> Scene Builder Kit is the core of the project, defines three main areas: 

Reviewer 2: ```suggestion
Scene Builder Kit is the core of the project and defines three main areas: 
```

- Left: Library of custom and built-in controls, Hierarchy and Controller of the FXML layout being edited
- Center: Workspace area for displaying the content of the FXML layout that is being designed
- Right: Inspector with properties, layout and event handlers of the components of the FXML layout.

> Kit is an API that allows the integration of these components and their functionallity with other applications or IDEs, being Scene Builder App the best example of such integration. Other basic example can be found here: [EmbeddedSceneBuilderDemo](https://github.com/gluonhq/EmbeddedSceneBuilderDemo). Also you can use in your projects the controls available in Scene Builder Kit.

Reviewer 2: ```suggestion
Scene Builder Kit contains an API that allows these components and their functionality to be integrated in other applications or IDEs. Scene Builder App is the best example of such integration. Another basic example can be found here: [EmbeddedSceneBuilderDemo](https://github.com/gluonhq/EmbeddedSceneBuilderDemo). You can also use the controls available in Scene Builder Kit in your project.
```

</dependency>
```

If you want to build and install the Scene Builder Kit in your local repository, run:

> `mvn clean install -f kit`

Reviewer 2: no `./mvnw` here?

Author: right
